core/src/components/checkbox/checkbox.ionic.scss

```diff
@@ -0,0 +1,166 @@
+  --border-color: #{$checkbox-ionic-icon-border-color-off};
+  --checkmark-width: 3;
+  --padding-top: #{$checkbox-ionic-padding-top};
+  --padding-bottom: #{$checkbox-ionic-padding-bottom};
+
+  // Background
+  --checkbox-background: #{$checkbox-ionic-icon-background-color-off};
+
+  // Transition
+  --transition: #{background $checkbox-ionic-transition-duration $checkbox-ionic-transition-easing};
+
+  // Size
+  --size: #{$checkbox-ionic-icon-size};
+  // add to existing selector
+
+  // margin is required to make room for outline on focus, otherwise the outline may get cut off
+  @include margin($checkbox-ionic-outline-width);  
+  @include padding(--padding-top, null, --padding-bottom, null);
+
+  // Target area
+  &::after {
+    @include position(50%, 0, null, 0);
+
+    position: absolute;
+
+    height: 100%;
+    min-height: 48px;
+
+    transform: translateY(-50%);
+
+    content: "";
+
+    cursor: pointer;
+
+    z-index: 1;
+  }
+
+  .native-wrapper{
+    position: relative;
+  }
+}
+
+// Ionic Design Checkbox Sizes
+// --------------------------------------------------
+:host(.checkbox-small) {
+  --padding-top: #{$checkbox-ionic-small-padding-top};
+  --padding-bottom: #{$checkbox-ionic-small-padding-bottom};
+
+   // Size
+   --size: #{$checkbox-ionic-small-icon-size};
+}
+
+// Ionic Design Checkbox Shapes
+// --------------------------------------------------
+:host(.checkbox-rectangular) {
+  --border-radius: #{$checkbox-ionic-rectangular-border};
+}
+
+// Ionic Design Checkbox Disabled
+// --------------------------------------------------
+// disabled, indeterminate checkbox
+:host(.checkbox-disabled.checkbox-indeterminate) .checkbox-icon {
+  border-width: 0;
+
+  background-color: #74aafc;
+}
+
+// disabled, unchecked checkbox
+:host(.checkbox-disabled) .checkbox-icon {
+  border-color: #c9c9c9;
+
+  background-color: #f5f5f5;
+}
+// disabled, checked checkbox
+:host(.checkbox-disabled.checkbox-checked) .checkbox-icon {
+  border-width: 0;
+
+  background-color: #A8C8F8;
+}
+
+// Ionic Design Checkbox Required State
+// --------------------------------------------------
+// Unhecked checkbox with `required` property set to true
+:host(.checkbox-required:not(.checkbox-checked):not(.checkbox-indeterminate)) {
+  .checkbox-icon {
+    border-color: #f72c2c;
+  }
+}
+
+// Focused: Unchecked checkbox with `required` property set to true
+:host(.ion-focusable.checkbox-required:not(.checkbox-checked):not(.checkbox-indeterminate)) .checkbox-icon {
+  outline-color: #ffafaf;
+}
+
+// Ionic Design Checkbox Focus Ring
+// --------------------------------------------------
+:host(.ion-focused:not(.checkbox-disabled)) .focus-ring {
+  @include position(-4px, -4px, -4px, -4px);
+  position: absolute;
+
+  width: calc(100% + 8px);
+  height: calc(100% + 8px);
+
+  transition: border-color 0.3s;
+
+  border: 2px solid $checkbox-ionic-focus-ring-color;
+
+  /* stylelint-disable-next-line property-disallowed-list */
```

Comment:
Ionic Framework provides a `@include border-radius` mixin. This is why we throw the stylelint error. 

This usage should be updated to:
```
@include border-radius(var(--border-radius));
```